### PR TITLE
Updated documentation

### PR DESCRIPTION
**Pull request by karmab**

### CODE REVIEW

**Comment by rthallisey** [comment]
@karmab I think leaving the serviceinstance template in ```templates``` makes more sense.  How come you want it in the ```launch``` directory?

**Comment by karmab** [comment]
@rthallisey the reason is that templates seems confusing for someone familiar with apbs. it looks like a reserved word. furthermore, the object in the templates directory has nothing to do with a template imho :)

**Comment by rthallisey** [comment]
> @rthallisey the reason is that templates seems confusing for someone familiar with apbs. it looks like a reserved word. furthermore, the object in the templates directory has nothing to do with a template imho :)
>
The CI the broker community uses the templates directory as a place to store a working service instance template.  I figured we would follow the same pattern.

**Comment by karmab** [comment]
my bad then

File: README.md [hunk 1]
```diff
@@ -1,22 +1,22 @@
...
+- *release* Whether to download the templates used from kubevirt github releases or render them from local templates
 
 ## How to use:
 
 #### Deploy from the UI
 
-1) Edit ```templates/kubevirt-apb.yml``` and add a ```admin_user``` and
-   ```admin_password``` that have cluster-admin permission.
-2) Deploy from command line.
+1) Edit ```launch/kubevirt-apb.yml``` and set ```admin_user``` and ```admin_password``` credentials that have cluster-admin permission.
```
Comment (rthallisey) [inline README.md]: ```launch``` -> ```templates```.

File: README.md [hunk 2]
```diff
@@ -1,22 +1,22 @@
...
-- release whether to download the templates used from kubevirt github releases or render them from local templates
-- openshift . whether to execute additional scc policy commands. Defaults to true
+- *tag*  Tag to use for kubevirt related images (defaults to 0.2.0)
+- *admin_user*
+- *admin_password*
+- *release* Whether to download the templates used from kubevirt github releases or render them from local templates
 
 ## How to use:
 
 #### Deploy from the UI
```
Comment (lukas-bednar) [inline README.md]: I know that this PR doesn't touch this part, but I think you can just fix this typo here.
I think it should be `Deploy from the CLI

**Pull request merged**